scrapy/resolver.py

```diff
@@ -4,17 +4,25 @@
 from scrapy.utils.datatypes import LocalCache
 
 # TODO: cache misses
-# TODO: make cache size a setting
 
 dnscache = LocalCache(10000)
 
 class CachingThreadedResolver(ThreadedResolver):
+    def __init__(self, reactor, settings):
```

Comment:
Are you asking this because of some options you want to add for https://github.com/scrapy/scrapy/pull/1104?